### PR TITLE
Adds the CIA, Removes UAAC-TIS

### PR DESCRIPTION
**Pull request by nauticall**
# About the pull request

Adds the CIA, the intelligence agency that needs to introduction. Featuring a briefcase full of goodies, covert operations, and several disguises, CIA agents can be planted into various places or even into the USCM.

This PR fully replaces the UAAC-TIS intelligence officers formerly in the game. Nobody really used them (save for a select few) and I figured it'd be better to go out with the American Gestapo and in with actual spies.

Note that this is admin spawn only for now. In the future it's possible additional covers, ERT or even survivor roles.

The following CIA covers have been added:

- Intelligence Analyst: The classic "tourist" business casual look of Vietnam-era CIA. Casual clothing, civilian rank, modifiable job.
- USCM Liaison: A (**real**) military officer who works with the CIA as a military liaison, though not in an outright public way. Can be either O-2 or O-3.
- Marine Raider: A CIA agent operating as a Marine Raider. They're on the same side, just possibly with differing objectives.
- UPP Soldier: A UPP grunt recruited as a CIA informant.
- UPP Senior Officer: A UPP _Starshiy Leytenant_ officer recruited as a CIA informant.
- CLF Engineer: A CLF field technician recruited as a CIA informant.

All of the above come with the CIA briefcase, a device capable of procuring various goods and weaponry for use in spycraft. The briefcase includes things like:
- Abduction and sedation weapons like chloroform, a tranq gun, and handcuffs.
- Basic stationery (pen and paper).
- Marine armor and clothing, for when shit hits the fan.
- Extra magazines.
- Hacking stuff.
- Radio keys for various factions, including the CIA's own radio channel.
- Listening bugs.
- A USCM codebook for verifying you are actually with the government, in case anyone has doubts.

# Explain why it's good for the game

Spycraft! I believe adding CIA adds a ton of roleplay potential with these people as well as the ability to carry out actual (admin-sanctioned) antaggery like abducting people or interrogating others in a subtle, government-sanctioned way.

# Things CIA Might Do
Event-runners use this table as your inspiration to find out what morally-questionable things your agents may do.

- Wiretap dialog in certain important areas like the CIC or brig.
- "Extract" an asset like a person or a thing from the Almayer or AO.
- Perform "enhanced interrogation methods".
- Subtly feed information or misinformation to command to sway them a little.
- Make a problematic person _disappear_.
- Local whistleblower found dead with 5 shots to the back of the head; ruled self-inflicted.

# The Elephant

CIA agents, naturally, work for the government of the United Americas, and nobody else. The USCM and CIA are fighting for the same team, but CIA might have interests or methods that differ slightly or are above the paygrades of most marines. 

CIA agents prioritize plausible deniability for covers with allies, and they're often well aware of the fact that people can see right through their generic cover identities; the goal isn't to trick allies, it's to prevent the CIA from being finger-pointed at directly if something goes south. The USCM knows there might be agents in their ranks, but since they're usually actually helpful - and pointing out a CIA agent does nothing for them anyway - they don't really care, and neither should marines. (Spies are a different story.)

For obvious reasons no CIA agent should ever say "I am with the CIA." If not outright disguised, it's up to the individual (or the admin) to fabricate a cover story, like "I'm an intelligence analyst working for ARMAT" or "I'm part of a battlefield survey team to improve logistics." You can modify your ID accordingly to suit your cover. Be creative, and be subtle.

# Screenshots
<details>
<summary>Screenshots & Videos</summary>

image
image
image
image
image

</details>

# Changelog
:cl:
add: Adds CIA agents to the game. These are disguised as either civilians, liaisons, or people in other factions and have a plethora of tools to achieve any objectives they might be assigned to.
del: Deletes the UAAC-TIS from the game.
/:cl:

### CODE REVIEW

**Comment by RoboticVodka** [comment]
Notably we do have the DNOP-CIU that fits the exact niche of the CIA per this old lore-tidbit if it's still considered canon: https://discord.com/channels/150315577943130112/942220424912203846/1007434833980555274 

I may also suggest if you stay with CIA though you put together an OWLF loadout as they're a part of the CIA. At least all the clothes exist (mask, hood, vest, and uniform) to make a full loadout just not weapons or any particular armor. Could give them some EMPs, some kinda of future-ish looking gun, motion detector, ETC and they'd probably be set.

File: code/datums/skills/uscm.dm [hunk 1]
```diff
@@ -503,3 +503,28 @@ SOLAR DEVILS (PvE Marines)
...
 		SKILL_PILOT = SKILL_PILOT_EXPERT,
 	)
+
+/*
+---------------------
+CIA
+---------------------
+*/
+
+/datum/skills/CIA
```
Comment (harryob) [inline code/datums/skills/uscm.dm:513]:
```suggestion
/datum/skills/cia
```

File: code/game/objects/items/devices/portable_vendor.dm [hunk 2]
```diff
@@ -199,6 +202,8 @@
 	update_overlays()
 
 /obj/item/device/portable_vendor/proc/update_overlays()
+	if(covert) return
```
Comment (harryob) [inline code/game/objects/items/devices/portable_vendor.dm:205]:
```suggestion
	if(covert)
		return
```

File: code/modules/gear_presets/cia.dm [hunk 3]
```diff
@@ -0,0 +1,198 @@
+/datum/equipment_preset/cia
+	name = "CIA"
+	flags = EQUIPMENT_PRESET_EXTRA
+	minimum_age = 25
+	assignment = JOB_CIA
+	skills = /datum/skills/CIA
```
Comment (harryob) [inline code/modules/gear_presets/cia.dm:6]:
```suggestion
	skills = /datum/skills/cia
```

File: code/modules/gear_presets/cia.dm [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+	new_human.equip_to_slot_or_del(new /obj/item/device/portable_vendor/antag/cia(new_human), WEAR_IN_BACK)
+
+/datum/equipment_preset/cia/officer/o3
+	name = "CIA Agent (USCM Liaison - Captain)"
+	paygrades = list(PAY_SHORT_MO3 = JOB_PLAYTIME_TIER_0)
+
+/datum/equipment_preset/uscm/marsoc/low_threat/cia
+	name = "CIA Agent (Marine Raider Plant)"
+	minimum_age = 30
+	skills = /datum/skills/CIA
```
Comment (harryob) [inline code/modules/gear_presets/cia.dm:94]:
```suggestion
	skills = /datum/skills/cia
```

File: code/modules/gear_presets/cia.dm [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+	//limbs
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/marine/knife(new_human), WEAR_FEET)
+	new_human.equip_to_slot_or_del(new /obj/item/clothing/gloves/marine, WEAR_HANDS)
+	//pockets
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/medical/socmed/not_op, WEAR_L_STORE)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/tools/tactical/full, WEAR_R_STORE)
+
+/datum/equipment_preset/clf/engineer/cia
+	name = "CIA Agent (CLF Engineer Disguise)"
+	skills = /datum/skills/CIA
```
Comment (harryob) [inline code/modules/gear_presets/cia.dm:139]:
```suggestion
	skills = /datum/skills/cia
```

File: code/modules/gear_presets/cia.dm [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+	new_human.equip_to_slot_or_del(new /obj/item/storage/pouch/firstaid/ert, WEAR_R_STORE)
+
+	spawn_rebel_weapon(new_human)
+	spawn_rebel_weapon(new_human,1)
+
+
+
+/datum/equipment_preset/upp/soldier/dressed/cia
+	name = "CIA Agent (UPP Soldier Disguise)"
+	skills = /datum/skills/CIA
```
Comment (harryob) [inline code/modules/gear_presets/cia.dm:175]:
```suggestion
	skills = /datum/skills/cia
```

File: code/modules/gear_presets/cia.dm [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+	access = get_access(ACCESS_LIST_CLF_BASE) + list(ACCESS_CIA)
+
+/datum/equipment_preset/upp/soldier/dressed/cia/load_gear(mob/living/carbon/human/new_human, client/mob_client)
+	. = ..()
+	new_human.equip_to_slot_or_del(new /obj/item/device/portable_vendor/antag/cia/covert, WEAR_IN_BACK)
+
+
+/datum/equipment_preset/upp/officer/senior/dressed/cia
+	name = "CIA Agent (UPP Senior Officer Disguise)"
+	skills = /datum/skills/CIA
```
Comment (harryob) [inline code/modules/gear_presets/cia.dm:188]:
```suggestion
	skills = /datum/skills/cia
```

File: code/modules/gear_presets/cia.dm [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+
+/datum/equipment_preset/clf/engineer/cia/New()
+	. = ..()
+	access = get_access(ACCESS_LIST_CLF_BASE) + list(ACCESS_CIA)
+
+/datum/equipment_preset/clf/engineer/cia/load_gear(mob/living/carbon/human/new_human, client/mob_client)
+	var/obj/item/clothing/under/colonist/clf/M = new()
+	var/obj/item/clothing/accessory/storage/webbing/W = new()
+	M.attach_accessory(new_human, W)
```
Comment (harryob) [inline code/modules/gear_presets/cia.dm:148]:
```suggestion
	var/obj/item/clothing/under/colonist/clf/uniform = new()
	var/obj/item/clothing/accessory/storage/webbing/webbing = new()
	uniform.attach_accessory(new_human, webbing)
```

File: code/modules/gear_presets/cia.dm [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+	new_human.equip_to_slot_or_del(M, WEAR_BODY)
```
Comment (harryob) [inline code/modules/gear_presets/cia.dm:149]:
```suggestion
	new_human.equip_to_slot_or_del(uniform, WEAR_BODY)
```

**Review by harryob — CHANGES_REQUESTED** [review]
goated

**Review by harryob — APPROVED** [review]

**Review by harryob — APPROVED** [review]

**Pull request merged**